ApprovalTests/integrations/catch/Catch2Approvals.h

```diff
@@ -4,6 +4,8 @@
 #include "ApprovalTests/namers/ApprovalTestNamer.h"
 #include "ApprovalTests/integrations/FrameworkIntegrations.h"
 
+#include <iostream>
```

Comment:
I did get a missing include statement for `std::cout` in this file when building locally, so I added the include.